src/ripple/app/misc/SHAMapStoreImp.h

```diff
@@ -84,6 +85,13 @@ class SHAMapStoreImp : public SHAMapStore
     static std::uint32_t const minimumDeletionInterval_ = 256;
     // minimum # of ledgers required for standalone mode.
     static std::uint32_t const minimumDeletionIntervalSA_ = 8;
+    // Ledger sequence at which the last deletion interval was triggered,
+    // or the current validated sequence as of first use
+    // if there have been no prior deletions. Deletion occurs up to (but
+    // not including) this value. All ledgers past this value are accumulated
+    // until the next online deletion. This value is persisted to SQLite
+    // nearly immediately after modification.
+    std::atomic<LedgerIndex> lastRotated_ {};
```

Comment:
Why does this need to be atomic? Maybe add to the comment explaining briefly why it needs to be atomic

Reply:
It's read from multiple threads, so it needs to be atomic. For synchronized things, atomics, things in mutexes, and such, I think that's generally pretty implicit.